Classes/Controllers/OpenRecentController.m

```diff
@@ -146,8 +145,8 @@ - (NSInteger)numberOfRowsInTableView:(NSTableView *)aTableView
 }
 
 - (IBAction)changeSelection:(id) sender {
-	int i = [resultViewer selectedRow];
-	if(i >= 0 && i < [currentResults count])
+	NSUInteger i = resultViewer.selectedRow;
+	if(i < currentResults.count)
```

Comment:
good catch, fixed.

Are we missing any compiler warnings there? I would’ve expected getting a warning/error for that.